migrations/3-create-menuitem.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('menuitem', {
```

Comment:
This should either be camelCase, or use under_scores. It's hard to read with alllowercase.